include/eve/constant/twotonmb.hpp

```diff
@@ -27,7 +27,7 @@ namespace eve
   //!
   //! | Member       | Effect                                                     |
   //! |:-------------|:-----------------------------------------------------------|
-  //! | `operator()` | Computes the twotonmb constant                               |
+  //! | `operator()` | Computes the twotonmb constant                             |
```

Comment:
Can't we just spell it out `the two to the nbmantissabits power`? It looks hyper self-referential in this state.

Reply:
done